### PR TITLE
Toyota: Move EPS torque factor to openpilot

### PR DESCRIPTION
**Pull request by sshane**
Previously the factors were defined in the DBC as well as here as the safety param. Now we read non-standard factors from a dict and use the default of `73` for all others. Also need to scale EPS torque in interface now since it's always 1 from the parser.

Needs:
- [x] https://github.com/commaai/opendbc/pull/522

### CODE REVIEW

File: selfdrive/car/toyota/values.py [hunk 1]
```diff
@@ -1684,6 +1685,8 @@ class CAR:
   CAR.ALPHARD_TSS2: dbc_dict('toyota_nodsu_pt_generated', 'toyota_tss2_adas'),
 }
 
+# These cars have non-standard EPS torque scale factors. All others are 73
+EPS_SCALE = defaultdict(lambda: 73, {CAR.PRIUS: 66, CAR.COROLLA: 88, CAR.LEXUS_IS: 77, CAR.LEXUS_RC: 77, CAR.LEXUS_CTH: 100})
```
Comment (adeebshihadeh) [inline selfdrive/car/toyota/values.py:1689]: just use a normal dict, then do `EPS_SCALE.get(car, 100)`

Comment (sshane) [reply selfdrive/car/toyota/values.py:1689]: Sure, just means we define it in two places if that's okay.

**Review by adeebshihadeh — APPROVED** [review]
LGTM. Feel free to merge after using a normal dict and bumping opendbc

File: selfdrive/car/toyota/values.py (hunk 1 above)
Comment (adeebshihadeh) [reply selfdrive/car/toyota/values.py]: actually, think you're right. the defaultdict is cleaner

**Comment by sshane** [comment]
Let's wait for tests and I'll merge

**Pull request merged**